### PR TITLE
A note about Cmd-Q

### PR DESCRIPTION
**Pull request by progapandist**
Too many people have no idea how to quit apps on Mac, here's an attempt to put a stop to this madness.

### CODE REVIEW

File: macOS.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ You can close Teamviewer now :wink:
 If you are not familiar with video calls, here is a great [article](https://martinfowler.com/articles/effective-video-calls.html) full of good practices :camera: :microphone:
 
 
+## A note about quitting apps on a Mac 
+
+Clicking the little red cross in the top left corner of the application window on a Mac **does not really quit it**, it just closes an active window. To quit the application _for real_ either press `Cmd + Q` when the application is active, or navigate to APP_NAME -> Quit in the menu bar.
```
Comment (db0sch) [inline macOS.md:43]: I'm not sure people know how to locate `APP_NAME -> Quit in the menu bar`
Shall we add a small screenshot?

Comment (progapandist) [reply macOS.md]: Done

**Comment by db0sch** [comment]
@progapandist Thanks for the screenshot.
I haven't noticed that when I commented, but you modified the final file `macos.md` which is generated by the script `build.rb`.
`build.rb` is concatenating a bunch of partials (cf the `partials` folder) into one document.
So, to keep your changes you have two choices:
- Either you include them into one of the partials that will be used by `build.rb`
- You create another partial like `osx_how_to_quit_programs.md` and you include it in the partial list in `build.rb`.

Finally, once you changed/added the parial, you have to re-run `ruby build.rb`, to regerate the end files and then commit the whole thing 😉

**Comment by progapandist** [comment]
Wow that sounds harsh. All right, I'll take care of it later then :)

**Comment by db0sch** [comment]
haha. way easier than you think.
do it when you have the time

**Comment by ssaunier** [comment]
⚠️ do not edit output file directly, as any edit not saved in `_partials` will be lost at next [build](https://github.com/lewagon/setup/blob/master/build.rb) phase.

**Comment by ssaunier** [comment]
Fixed by 24f71f6881d0010b9f2fef851c1aaf41b603223e

**Pull request closed without merging**